### PR TITLE
Adding inception model

### PR DESCRIPTION
**Pull request by blclo**
Added inception model and some small fixes/aids that also fix the problems I encountered.

### CODE REVIEW

File: src/models/train.py
Comment (albertkjoller) [inline src/models/train.py:1]: Not sure why we need the logits? In my opinion it would be nicer if the training loop is free of code that is model-dependent - that was why I put the "get_model", etc. in another script

Comment (albertkjoller) [reply src/models/train.py:1]: Lines: 71-73 and 99-101

Comment (albertkjoller) [inline src/models/train.py:1]: Also interpolation in line 92-94 seems a bit overkill - maybe we should just run the data-processing with shape 299 instead ?

**Review by albertkjoller — CHANGES_REQUESTED** [review]
